### ISSUE TITLE
Blog post is gone

### ISSUE THREAD

**Issue by arnonuem** [tendermint/abci#38]
The referenced blog post in readme.md is gone!

"For more information on TMSP, motivations, and tutorials, please visit our blog post."

Please update the link or archive the content of the blog post.

**Comment by jorpic** [comment]
It is still gone.
The page is blank and there is this error in JS console:
```
vue.runtime.common.js:3105  Uncaught TypeError: Cannot read property 'title' of undefined
```

**Comment by melekes** [comment]
Sorry for that guys. Will be fixed soon in #49.
